app/classes/pattern_search/base.rb

```diff
@@ -9,7 +9,13 @@ def initialize(string)
       self.errors = []
       self.parser = PatternSearch::Parser.new(string)
       build_query
+      real_model = model.name.to_sym
       self.query = Query.lookup(model.name.to_sym, args)
```

Comment:
Is line 13  needed (`self.query = Query.lookup(model.name.to_sym, args`)
in light of new line 18 (`self.query = Query.lookup(real_model, args`) ?

Reply:
Good catch.  Fat fingered that one.